tests/functional/adapter/test_constraints.py

```diff
@@ -13,11 +13,16 @@
     id integer not null primary key ,
     color text ,
     date_day date
-) as (
+) as ( select
```

Comment:
Maybe I'm reading this wrong, but I don't think this change does anything.

Reply:
In that case the order of the columns was the same in SQL and yaml. I changed the test case to make the difference more obvious.

Reply:
Ah, column order, got it.